packages/smooth_app/lib/pages/product/product_page.dart

```diff
@@ -470,17 +475,15 @@ class _ProductPageState extends State<ProductPage> {
       // no list to add to
       return;
     }
-    final dynamic target = await showModalBottomSheet<dynamic>(
+    final dynamic target = await showCupertinoModalBottomSheet<dynamic>(
```

Comment:
One day we should state which one to use between `showModalBottomSheet` and `showCupertinoModalBottomSheet`, and in which cases.
We'll see that later.

Reply:
I just checked which one we currently use more, `showModalBottomSheet` we only used once and `showCupertinoModalBottomSheet` several times, I just changed it to make it consistent, Are there any noticable differences between them